plugins/chunter-resources/src/components/ChannelHeader.svelte

```diff
@@ -13,37 +13,43 @@
 // limitations under the License.
 -->
 <script lang="ts">
-  import { Channel } from '@anticrm/chunter'
+  import { Channel, DirectMessage } from '@anticrm/chunter'
```

Comment:
We should have different space headers (look at slack), you can make empty component and fill it in separate PR

Reply:
Done